### PR TITLE
translation english and spanish

### PR DESCRIPTION
**Pull request by jeann2013**
**Describe Pull request**
Add translation English and Spanish

### CODE REVIEW

File: locales/es.lua [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    error = {
+        you_have_clocked_out = "Has registrado la salida"
+    },
+}
+
+Lang = Locale:new({
+    phrases = Translations,
+    warnOnMissing = true
+})
```
Comment (tom-osborne) [inline locales/es.lua:27]: Change this to use the convar setting as follows: - 
```lua
if GetConvar('qb_locale', 'en') == 'es' then
    Lang = Locale:new({
        phrases = Translations,
        warnOnMissing = true,
        fallbackLang = Lang,
    })
end
```

File: locales/en.lua [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    error = {
+        you_have_clocked_out = "You Have Clocked Out"
+    },
+}
+
+Lang = Locale:new({
+    phrases = Translations,
+    warnOnMissing = true
+})
```
Comment (tom-osborne) [inline locales/en.lua:27]: This needs to be as follows: - 
```lua
Lang = Lang or Locale:new({
    phrases = Translations,
    warnOnMissing = true
})
```

File: fxmanifest.lua [hunk 3]
```diff
@@ -4,7 +4,11 @@ game 'gta5'
 description 'QB-RecycleJob'
 version '2.1.0'
 
-shared_script 'config.lua'
+shared_scripts {
+  'config.lua',
+  '@qb-core/shared/locale.lua',
+  'locales/en.lua', -- Change to the language you want
+}
```
Comment (tom-osborne) [inline fxmanifest.lua:11]: Please change this as follows: 
```lua
shared_scripts {
    '@qb-core/shared/locale.lua',
    'locales/en.lua',
    'locales/*.lua',
    'config.lua'
}
```

**Review by tom-osborne — CHANGES_REQUESTED** [review]

File: fxmanifest.lua (hunk 3 above)
Comment (jeann2013) [reply fxmanifest.lua:12]: done

File: locales/en.lua (hunk 2 above)
Comment (jeann2013) [reply locales/en.lua:27]: done

File: locales/es.lua (hunk 1 above)
Comment (jeann2013) [reply locales/es.lua]: done

**Comment by jeann2013** [comment]
@tom-osborne fixed

**Review by tom-osborne — APPROVED** [review]

**Pull request merged**